WooCommerce/Classes/ViewRelated/Inbox/InboxNoteRow.swift

```diff
@@ -40,7 +39,19 @@ struct InboxNoteRow: View {
                     // HStack with actions and dismiss action.
                     HStack(spacing: Constants.spacingBetweenActions) {
                         ForEach(viewModel.actions) { action in
-                            if action.url != nil {
+                            if viewModel.isSurvey {
+                                Button(action.title) {
+                                    withAnimation(.easeInOut) {
+                                        surveyCompleted = true
+                                    }
+                                    viewModel.markInboxNoteAsActioned(actionID: action.id)
+                                }
+                                .buttonStyle(SecondaryButtonStyle())
+                                .frame(minWidth: Constants.minWidthSurveyButton)
```

Comment:
❓  I wonder why we need `minWidth` for the buttons? Would it work for other languages with different lengths for the content? How about using a `Text` with padding for each button's title?

Reply:
After a discussion with the design team, we have decided to keep a minimum button width of 40px. We discussed this [here](https://a8c.slack.com/archives/C02N7N92DEV/p1646267961802689?thread_ts=1645771532.573489&cid=C02N7N92DEV). No issues with other languages and different lengths, since this is the minimum width of the button (so, a lower bound). They may widen further if the texts are longer (you can check this case in the SwiftUI preview).